src/add-hooks.coffee

```diff
@@ -12,6 +12,8 @@ sandboxHooksCode = require './sandbox-hooks-code'
 mergeSandboxedHooks = require './merge-sandboxed-hooks'
 HooksWorkerClient = require './hooks-worker-client'
 
+# Ensure platform agnostic path.basename function
+basename = if process.platform is 'win32' then path.win32.basename else path.basename
```

Comment:
AppVeyor failed for this PR, so I guess this doesn't work. When I've seen AppVeyor failing, I was afraid we will need to do some hairy debugging, but reading the code, the problem needs to be on this line, so luckily this is pretty isolated and should be easy to track down :)